src/tools/interop/idt/capture/pcap/pcap.py

```diff
@@ -0,0 +1,62 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import os
+import time
+
+from capture.file_utils import create_standard_log_name
+from capture.shell_utils import BashRunner
+
+
+class PacketCaptureRunner:
```

Comment:
How will this extent to packet capture on endpoint device (e.g. rasperry pi running all-clusters-app, with tcpdump) or on Android phone, or on hub (e.g. Echo, with tcpdump)? I think the "capture" interface might have `start`, `stop`, and `setup` and `teardown`. Some of the analyzers that we want to configure will need to be pushed to the device and maybe some cleanup actions.

Reply:
Thanks! I suppose the idea for the packet capture implementation here (in the root of the package) was only for the capture / discovery execution, where things like echo and android would be implemented in `platform`. How do you feel about that organization? It could of course still make sense to have a shared interface in that structure?

Reply:
Can you please create an interface for this module?